### ISSUE TITLE
[bug] Crashes when opening the "Sorting" section of the new config

### ISSUE THREAD

**Issue by GhostIsBeHere** [Siphalor/mouse-wheelie#138]
**Versions**
1.16.5 1.7.3-newconfig.2 (Beta)
1.17.1 (newconfig)
**Describe the bug**
MouseWheelie crashes Minecraft when opening the "Sorting" Section of the new config

**To Reproduce**
1. Open game
2. Open Mod Menu
3. Open Mouse Wheelie config
4. Click on sorting

**Expected behavior**
I would be able to change the sorting config

**Log**
Unable to find anything important - Here is a very long log, scroll to the bottom and look at everything from "[15/08/2021 17:12:29 PM] Preparing for launch!" to end. <https://paste.atlauncher.com/view/dda04c95-2a9c-439c-bb70-018bf50b3667>

**Comment by RasmusKD** [comment]
happens in 1.17.1 too

**Comment by blackd** [comment]
Happens in 1.17.1 with this notable exception in the logs

```
java.lang.AbstractMethodError: Method de/siphalor/tweed4/tailor/coat/entry/CoatDropdownSelectInput.method_1865()V is abstract
	at de.siphalor.tweed4.tailor.coat.entry.CoatDropdownSelectInput.method_1865(CoatDropdownSelectInput.java)
	at de.siphalor.coat.list.entry.ConfigListConfigEntry.tick(ConfigListConfigEntry.java:157)
	at de.siphalor.coat.list.DynamicEntryListWidget.tick(DynamicEntryListWidget.java:641)
	at de.siphalor.coat.screen.ConfigScreen.method_25393(ConfigScreen.java:246)
	at net.minecraft.class_310.method_1572(class_310.java:1720)
	at net.minecraft.class_437.method_25412(class_437.java:490)
	at net.minecraft.class_310.method_1574(class_310.java:1720)
	at net.minecraft.class_310.method_1523(class_310.java:1081)
	at net.minecraft.class_310.method_1514(class_310.java:728)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
```

**Comment by Siphalor** [comment]
Should be fixed in 1.7.3-newconfig.3 🙂
